test/integration/targets/async/tasks/main.yml

```
@@ -143,6 +147,7 @@
     - async_result.ansible_job_id is match('j\d+\.\d+')
     - async_result.finished == 1
     - async_result is finished
+    - async_result.changed == 0
     - async_result.changed == false
     - async_result is not changed
     - async_result.failed == true
```

This can be simplified:

```suggestion
    - async_result.finished
    - async_result.changed is false
    - async_result.failed
```

Any necessary tests for the test plugins (started, finished, changed) should be in the test_core integration test. The existing tests there may already cover those test plugins adequately.